Geometry/TrackerGeometryBuilder/python/trackerDigiGeometryESModule_cfi.py

```diff
@@ -0,0 +1,18 @@
+import FWCore.ParameterSet.Config as cms
+
+## TEMPLATE as it used in the HLT menu
+trackerDigiGeometryESModule = cms.ESProducer( "TrackerDigiGeometryESModule",
+  appendToDataLabel = cms.string( "" ),
+  fromDDD = cms.bool( False ),
+  trackerGeometryConstants = cms.PSet(
+    ROCS_X = cms.int32( 0 ),
+    ROCS_Y = cms.int32( 0 ),
+    upgradeGeometry = cms.bool( False ),
+    BIG_PIX_PER_ROC_Y = cms.int32( 2 ),
+    BIG_PIX_PER_ROC_X = cms.int32( 1 ),
+    ROWS_PER_ROC = cms.int32( 80 ),
+    COLS_PER_ROC = cms.int32( 52 )
+  ),
+  applyAlignment = cms.bool( True ),
+  alignmentsLabel = cms.string( "" )
+)
```

Comment:
This file is not missing as it is auto-generated. See 
http://cmslxr.fnal.gov/source/Geometry/TrackerGeometryBuilder/plugins/TrackerDigiGeometryESModule.cc?v=CMSSW_7_4_X_2015-01-15-0200#0049
This newly introduced file could be a reason for RelVal failures.